front_end/src/components/booking/BookingDetailsCard.js

```diff
@@ -8,6 +8,7 @@ import Button from "@material-ui/core/Button";
 import Typography from "@material-ui/core/Typography";
 import lightBlue from "@material-ui/core/colors/lightBlue";
 import SvgIcon from "@material-ui/core/SvgIcon";
+import { connect } from "react-redux";
```

Comment:
are you even using this file?

Reply:
I'm not referring to line 11, but the file as a whole

Reply:
removed the files